Source/WTF/wtf/text/TextBreakIterator.h

```diff
@@ -181,13 +182,13 @@ class CachedTextBreakIterator {
     WTF_MAKE_FAST_ALLOCATED;
 public:
     CachedTextBreakIterator(StringView string, const UChar* priorContext, unsigned priorContextLength, TextBreakIterator::Mode mode, const AtomString& locale, TextBreakIterator::ContentAnalysis contentAnalysis = TextBreakIterator::ContentAnalysis::Mechanical)
-        : m_backing(TextBreakIteratorCache::singleton().take(string, priorContext, priorContextLength, mode, contentAnalysis, locale))
```

Comment:
Should we add Debug assertions in `TextBreakIteratorCache::singleton().take()` and `put()` that they are always called on the main thread?  (This doesn't block landing, but it would catch issues quicker in the future.)